composer.json

```diff
@@ -12,7 +12,7 @@
         }
     ],
     "require": {
-        "php": "^7.0",
+        "php": "^7.0 | ^8.0",
```

Comment:
Use `||` as per https://github.com/ruflin/Elastica/blob/master/composer.json#L18 for consistency